### PR TITLE
 🐛 ✨ Update openapi specs and expose studies api in dev mode

### PR DESCRIPTION
**Pull request by bisgaard-itis**
## What do these changes do?
- Fix download file bug by updating openapi.json
- Expose studies api in dev mode.

## Related issue/s
- resolves https://github.com/ITISFoundation/osparc-issues/issues/1364

### CODE REVIEW

File: clients/python/client/osparc/_studies_api.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
+from osparc_client import StudiesApi as _StudiesApi
+
+from ._utils import dev_feature
+
+
+class StudiesApi(_StudiesApi):
+    """Class for interacting with solvers"""
+
+    @dev_feature
+    def clone_study(self, study_id, **kwargs):
```
Comment (wvangeit) [inline clients/python/client/osparc/_studies_api.py:10]: For me this is ok for now, but wouldn't it be better to automate this?
(Or one could get all the methods from super, or if one wants to limit it, just provide a list with names of methods, and not writing all the procedural code?)
If this automation already exists, and this file is a result of it, I would not commit the resulting file in the repo.

**Review by wvangeit — APPROVED** [review]
Thx, looks ok to me, but just seem my comment

File: clients/python/client/osparc/_studies_api.py (hunk 1 above)
Comment (bisgaard-itis) [reply clients/python/client/osparc/_studies_api.py]: Good point. I was too lazy to do it, but I have fixed that now :smile

Comment (wvangeit) [reply clients/python/client/osparc/_studies_api.py]: Ok, great. Lazy is good, but I'm mostly too lazy to do you first approach  😄

**Pull request merged**